parsers/GB_NIR.py

```diff
@@ -204,27 +179,6 @@ def IE_processor(df):
     return datapoints
 
 
-def merge_production(thermal_data, wind_data):
-    """
-    Joins thermal and wind production data on shared datetime key.
-    Returns a list.
-    """
-
-    total_production = thermal_data + wind_data
-
-    # Join thermal and wind dicts on 'datetime' key.
-    d = defaultdict(dict)
-    for elem in total_production:
-        d[elem['datetime']].update(elem)
-
-    joined_data = sorted(d.values(), key=itemgetter("datetime"))
-
-    for datapoint in joined_data:
-        datapoint['datetime'] = add_default_tz(parser.parse(datapoint['datetime'], dayfirst=True))
```

Comment:
this line needs to be used L131 in `production_processor` in order to properly format `datetime`

Reply:
Ah, thanks Oli! :)
Now the datetime looks good:

`fetch_production() ->
[{'zoneKey': 'GB-NIR', 'datetime': datetime.datetime(2019, 4, 16, 0, 0, tzinfo=tzfile('GB-Eire')), 'production': {'coal': 0.0, 'gas': 365.526, 'oil': 0.0, 'solar': None, 'wind': 385.647}, 'source': 'soni.ltd.uk'}, {'zoneKey': 'GB-NIR', 'datetime': datetime.datetime(2019, 4, 16, 0, 15, tzinfo=tzfile('GB-Eire')), 'production': {'coal': 0.0, 'gas': 363.186, 'oil': 0.0, 'solar': None, 'wind': 377.49300000000005}, 'source': 'soni.ltd.uk'}`